### PR TITLE
 Added numeric datatype handling

### PR DESCRIPTION
**Pull request by pwilken**
The numeric datatype was not handled, is now handled like the decimal datatype.

http://www.firebirdfaq.org/faq340/
"numeric = decimal" in the firebird world

### CODE REVIEW

**Comment by ralmsdeveloper** [comment]
I will implement unit tests, so the new PR's are validated!

File: EFCore.FirebirdSql/Storage/Internal/Mapping/FbTypeMapper.cs [hunk 1]
```diff
@@ -53,6 +53,7 @@ public class FbTypeMapper : RelationalTypeMapper
         private readonly FloatTypeMapping _float = new FloatTypeMapping("FLOAT");
         private readonly DoubleTypeMapping _double = new DoubleTypeMapping("DOUBLE PRECISION");
         private readonly DecimalTypeMapping _decimal = new DecimalTypeMapping($"DECIMAL({DefaultDecimalPrecision},{DefaultDecimalScale})");
+        private readonly DecimalTypeMapping _numeric = new DecimalTypeMapping($"NUMERIC({DefaultDecimalPrecision},{DefaultDecimalScale})");
```
Comment (ralmsdeveloper) [inline EFCore.FirebirdSql/Storage/Internal/Mapping/FbTypeMapper.cs]: Where is this being used?

Comment (pwilken) [reply EFCore.FirebirdSql/Storage/Internal/Mapping/FbTypeMapper.cs]: If a firebird database table has a column of the type numeric. 

{ "NUMERIC", _decimal },

**Comment by ralmsdeveloper** [comment]
Thanks for contribution!

**Pull request merged**